### DISCUSSION TITLE
Is there a way to change the skin of the bot in mineflayer?

### DISCUSSION THREAD

**Discussion by avacadox21** [PrismarineJS/mineflayer#1900] [General]
## Hey, I wanted to know if I can change the skin of the bot. 
I need to know about fast, it is urgent!

**Comment by aesthetic0001** [comment]
yes you can change from your minecraft account

**Comment by aesthetic0001** [comment]
please ask questions like these in discussions or the discord (https://discord.gg/ZKCFhSTK)
